.gitattributes

```diff
@@ -0,0 +1,32 @@
+# This file mainly controls line ending conversion behaviour, if
+# the user has the setting core.autocrlf true.
+
+# The meaning of the attributes is a little odd
+# -crlf means DO NOT convert line endings
+# crlf means CONVERT to lf in the repo & Linux/Mac, crlf on Windows
```

Comment:
This comment is not correct according to the git documentation (https://git-scm.com/docs/gitattributes), it should be something like:

```
# crlf means CONVERT to lf in the repo, but always checkout using crlf on all platforms
```